src/api/ApiContext.ts

```diff
@@ -1,20 +1,20 @@
 import { ApiPromise } from '@polkadot/api'
 import { Context, Consumer, Provider, createContext } from 'react'
-import { Chain } from '../chains'
-import { IRpcClient } from './chainApi/rpcClient'
+import { Chain, INFTAdapter, IRpcClient } from './chainApi/types'
 
 export type ChainData = {
   properties: {
     tokenSymbol: string
   }
-  systemChain: string;
-  systemName: string;
+  systemChain: string
+  systemName: string
 }
 
 export type ApiContextProps = {
   rpc: IRpcClient
-  rpcApi?: ApiPromise,
-  chainData?: ChainData,
+  rpcApi?: ApiPromise
```

Comment:
Maybe we don't need rawRpc here, because it is now private property? Actually, if we need some new action with raw api we must implement a new method in the controller. Or not?